### PR TITLE
Endpoints fix

### PR DESCRIPTION
**Pull request by tomdevman**
**DESCRIPTION:**
This PR addresses a fix for the orders service add/removing items now using the stock service (rather than using the stock table for retrieving this information)

**AUTHORS:**
@cldme

### CODE REVIEW

File: services/orders/item_add/lambda_function.py [hunk 1]
```diff
@@ -8,20 +8,47 @@
 dynamodb = boto3.resource('dynamodb')
 # get the users table
 ORDERS_TABLE = os.environ['ORDERS_TABLE']
-STOCK_TABLE = os.environ['STOCK_TABLE']
+
+# invoke lambda function with given name and payload
+def invoke_lambda(name, payload, invocation_type='RequestResponse'):
+    print(f'invoking lambda function: {name}')
+    client = boto3.client('lambda')
```
Comment (fabianishere) [inline services/orders/item_add/lambda_function.py]: This client could be shared between requests by initiating it globally, thus saving some overhead.

**Review by fabianishere — COMMENTED** [review]
Looks okay to me! I added one comment, please see if you want to address it, otherwise we can go ahead and merge this.

File: services/orders/item_add/lambda_function.py (hunk 1 above)
Comment (jwbambacht) [reply services/orders/item_add/lambda_function.py]: Agree if the implementation is not too complex.

Comment (fabianishere) [reply services/orders/item_add/lambda_function.py]: As far as I know, the implementation should be a matter of moving the assignment out of the function.

**Review by jwbambacht — APPROVED** [review]
Looks good!

**Comment by fabianishere** [comment]
@cldme If you need help on this, let us know!

File: services/orders/item_add/lambda_function.py (hunk 1 above)
Comment (tomdevman) [reply services/orders/item_add/lambda_function.py]: done !

**Pull request merged**